test/integration/src/test/resources/controller.config.properties

```diff
@@ -0,0 +1,11 @@
+###
```

Comment:
why is is this file needed? Is it actually used somewhere? 

There should be no Controller at all.